cfme/tests/infrastructure/test_provisioning_dialog.py

```diff
@@ -36,6 +36,8 @@ def cleanup_vm(vm_name, provider_key, provider_mgmt):
 def pytest_generate_tests(metafunc):
     # Filter out providers without provisioning data or hosts defined
     argnames, argvalues, idlist = testgen.infra_providers(metafunc, 'provisioning')
+    if metafunc.function.func_name == 'test_disk_format_select':
```

Comment:
I thought about that too, they are highly related, but we could have a folder in place that just contains all the test_provisioning_dialog test files. It really depends if we want to nest that far. I'm happy doing it like this for now as it's not going to be something that happens often, I hope!